client/jetpack-connect/auth-logged-in-form.jsx

```diff
@@ -470,7 +470,14 @@ class LoggedInForm extends Component {
 	}
 
 	getRedirectionTarget() {
-		return PLANS_PAGE + this.props.siteSlug;
+		const { partnerId, siteSlug } = this.props;
+
+		// Redirect sites hosted on Pressable with a partner plan to some URL.
+		// 51652 is a testing partner ID.
+		if ( 49640 === partnerId || 51652 === partnerId ) {
```

Comment:
Can we move those magic numbers to a constant somewhere ( for example, https://github.com/Automattic/wp-calypso/blob/09997d96d08a276df3e410eb5c93417b312dc6c2/client/jetpack-connect/auth-logged-in-form.jsx#L61  ) labeling as what they are? :)